### ISSUE TITLE
DietPi-Software | Jessie: qBittorrent login error

### ISSUE THREAD

**Issue by Tnnuy-sv** [MichaIng/DietPi#1499]
I have the same problem! and already tried to change how and said ... I have the XU4 odroid, but already on the raspberry pi 3 does not happen!! Does anyone know how to solve the problem?

<a href="http://pt-br.tinypic.com?ref=2zr2muo" target="_blank"><img src="http://i68.tinypic.com/2zr2muo.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

**Comment by Fourdee** [comment]
@tnnuyri 

thanks for the report 👍 
Looks limited to Debian Jessie. We'll need to investigate, leave it with us.

**Comment by Fourdee** [comment]
Notes:
```
WebUI\Password_ha1=@ByteArray(90c322f4dcabce6e66e8ea8aff218775)
WebUI\Username=root
```

Reverse hash:
https://isc.sans.edu/tools/reversehash.html
md5 hash 90c322f4dcabce6e66e8ea8aff218775 = Sorry, no solution found.

🈯️ https://md5hashing.net/hash/md5/90c322f4dcabce6e66e8ea8aff218775 ```dietpi```

---

https://packages.debian.org/jessie/qbittorrent-nox

**Comment by ghost** [comment]
I posted an issue with this problem around a month ago. I ended up just switching to using Transmission instead of qBittorrent, and i actually think that transmission is better. Worked flawlessly for me, nice and functional UI and it works with external drives.

**Comment by Tnnuy-sv** [comment]
Thanks for the answer! 
n my point of view I think the qBittorrent better !!
I've used it on my raspberry pi 3

Thank you for the excellent dietpi project

**Comment by symbios24** [comment]
UnfairGamer how did you mound external drives to Transmission? 
you just plug the drive and it founds it right away?

**Comment by ghost** [comment]
symbios, i thought it had detected it myself, but when i checked now, i have the fstab auto-mounting the disk to a folder. So i just have it saved in that "folder" that is actually the disk. Search linux fstab mount on boot

**Comment by Fourdee** [comment]
Seems Jessie version does not like/support md5sum generated, Stretch does:
```
WebUI\Password_ha1=@ByteArray(xxxx)
```
https://github.com/qbittorrent/qBittorrent/wiki/Web-UI-password-locked-on-qBittorrent-NO-X-%28qbittorrent-nox%29
even ```secret``` does not work when set, only removing the line and using ```adminadmin``` is functional.

---

Setting ```dietpi``` password via UI saves following to config:
```
WebUI\Password_ha1=@ByteArray(5e434b0004be6970cf8878fcf929baba)
```
Which does not match md5sum generation (that works on stretch qbittorrent-nox
```
root@DietPi:~# echo -ne 'dietpi' | md5sum
90c322f4dcabce6e66e8ea8aff218775
```

**Comment by Fourdee** [comment]
### Workaround

```
sed -i '/Password_ha1=/d' "$HOME"/.config/qBittorrent/qBittorrent.conf
dietpi-services restart
```

Password is now ```adminadmin```

**Comment by Fourdee** [comment]
Resolved for v6.2, online doc updated to reflect password change for Jessie:
http://dietpi.com/phpbb/viewtopic.php?f=8&t=5&p=2272#p2272

Install verified.

Completed.

**Comment by Tnnuy-sv** [comment]
thank you! Problem solved ... after and just enter and put the password that you want
